Disher/View Controllers/RecipesViewController.m

```diff
@@ -31,8 +31,10 @@ @interface RecipesViewController () <UITableViewDelegate, UITableViewDataSource,
 @property (nonatomic, strong) NSArray<Recipe *> *filteredTableViewRecipes;
 
 @property (nonatomic, strong) NSArray *spoonacularIngredientsInfo;
-@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString*> *mealDBMatches;
-@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString*> *spoonacularMatches;
+@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString *> *mealDBMatches;
+@property (nonatomic, strong) NSMutableDictionary<NSString *, NSNumber *> *mealDBMatchesValues;
+@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString *> *spoonacularMatches;
+@property (nonatomic, strong) NSMutableDictionary<NSString *, NSNumber *> *spoonacularMatchesValues;
```

Comment:
Can we make these names more descriptive? Maybe something like `mealDBMatchDescriptors` and `mealDBMatchPercentages`